src/app/Instance/InstancesListPage/InstancesListPage.tsx

```diff
@@ -26,99 +30,140 @@ const InstancesListPage = () => {
       status: "ready",
+      submitted_at: "2022-02-05T13:58:00Z",
     },
     {
-      name: "Instance six",
+      description: "Description for the instance six",
       id: "d7e13602-b046-4120-b377-15d61e21c31a",
-      submitted_at: "2022-02-01T12:02:00Z",
+      name: "Instance six",
       status: "ready",
+      submitted_at: "2022-02-01T12:02:00Z",
     },
     {
-      name: "Instance seven",
+      description: "Description for the instance seven",
       id: "3a7efbed-3562-4a95-9c32-f49d12d8cab2",
-      submitted_at: "2021-12-25T21:46:00Z",
+      name: "Instance seven",
       status: "ready",
+      submitted_at: "2021-12-25T21:46:00Z",
     },
     {
-      name: "Instance eight",
+      description: "Description for the instance eight",
       id: "fa648473-3662-4bbc-99ba-158d9ab95ccc",
-      submitted_at: "2021-12-20T12:21:00Z",
+      name: "Instance eight",
       status: "ready",
+      submitted_at: "2021-12-20T12:21:00Z",
     },
     {
-      name: "Instance nine",
+      description: "Description for the instance nine",
       id: "04f131c3-b34c-4ee2-b153-fbff0bb91ece",
-      submitted_at: "2021-12-15T16:09:00Z",
+      name: "Instance nine",
       status: "ready",
+      submitted_at: "2021-12-15T16:09:00Z",
     },
     {
-      name: "Instance ten",
+      description: "Description for the instance ten",
       id: "c28da8c0-05e4-42f3-a3fd-615cc7fbb382",
-      submitted_at: "2021-12-10T11:34:00Z",
+      name: "Instance ten",
       status: "ready",
+      submitted_at: "2021-12-10T11:34:00Z",
     },
     {
-      name: "Instance eleven",
+      description: "Description for the instance eleven",
       id: "a8c1cb57-0ab3-4ccb-8c55-ef8a8b166846",
-      submitted_at: "2021-12-05T11:34:00Z",
+      name: "Instance eleven",
       status: "ready",
+      submitted_at: "2021-12-05T11:34:00Z",
     },
     {
-      name: "Instance twelve",
+      description: "Description for the instance twelve",
       id: "e176d63b-6fdc-43ce-afc8-45160f456502",
-      submitted_at: "2021-12-01T11:34:00Z",
+      name: "Instance twelve",
       status: "ready",
+      submitted_at: "2021-12-01T11:34:00Z",
     },
   ];
-  const { t } = useTranslation("openbridgeTempDictionary");
 
-  //TODO fake actionResolver
-  const actionResolver = (rowData: IRowData) => {
-    // noinspection JSUnusedGlobalSymbols
+  const actionResolver = () => {
     return [
       {
-        title: "Action",
-        onClick: () => console.log(rowData),
+        title: t("common.details"),
+        onClick: () =>
+          // @TODO missing action to perform when clicking on details action
+          {},
+      },
+      {
+        title: t("common.delete"),
+        onClick: () =>
+          // @TODO missing action to perform when clicking on delete action
+          {},
       },
     ];
   };
 
   const [showCreateInstance, setShowCreateInstance] = useState(false);
 
+  const getPagination = (isCompact: boolean, customStyle?: CSSProperties) => (
```

Comment:
Minor observation given this will change. I would probably pass the `itemCount` too as param.